Makefile

```diff
@@ -18,7 +18,16 @@ all: build
 -include $(CONFIG)
 -include $(CONFIG)-token
 
-DOCKER_REPO ?= k8s-publishing-bot
+GIT_TAG ?= $(shell git describe --tags --always --dirty)
+
+# Image variables
+IMG_REGISTRY ?= gcr.io/k8s-staging-publishing-bot
+IMG_NAME = publishing-bot
+
+IMG_VERSION ?= v0.0.0-1
+
+# TODO(image): Consider renaming this variable
+DOCKER_REPO ?= $(IMG_REGISTRY)/$(IMG_NAME)
```

Comment:
I'm going to hold off on renaming this until we start promoting images to production, otherwise the `sed` deploy logic will fall over (because it's looking for a `latest` tag).